stylesheet.css

```diff
@@ -0,0 +1,187 @@
+body{
+	background-color: #f8f8f8;
+	margin: 0px;
+}
+#navbar{
+	display: flex;
+	background-color: white;
+	height: 56px;
+    padding: 0 16px;
+   
+    align-items: center;
+    /*justify-content: space-between;*/
```

Comment:
Consider deleting commented out codes. If you need to review a previous version of your code, use git history.

Please do this for all other occurrences.

Reply:
Thanks for your review. All comments has been removed.